aries-site/src/pages/components/namevaluelist.mdx

```diff
@@ -202,6 +207,92 @@ For vertical lists:
   <NameValueListReadMoreExample />
 </Example>
 
+## Name-Value Pair with Empty Value
+
+If the value of a name-value pair is empty, use `--` in place of where a value would appear. This visual indicator aids the user in identifying fields that do not have values without increasing cognitive load.
+
+Add an `a11yTitle="No value"` to the Text component that would otherwise render the data to ensure users of assisstive technology are informed of the empty state.
```

Comment:
```suggestion
To inform assistive-technology users of an empty value, add an `a11yTitle="No value"` to the Text component that would otherwise render the data.
```
I'm always torn between presenting the action first or the condition in which it's used but I default to the latter for slightly more complicated sentences. The original still works though so this is 100% your call.

Reply:
☝️ I prefer @CitizenFox's approach. I too like to follow the format which leads with "what is being accomplished" followed by "how to achieve."